src/constantProductCalculator.ts

```diff
@@ -1,8 +1,25 @@
 import D from "decimal.js";
 
+import { isqrt } from "./isqrt";
 import { Pool } from "./pool";
 import { SwapCalculator } from "./types";
 
+export function getConstantProductMintedLiquidityTokens(
+  addedPrimary: bigint,
+  addedSecondary: bigint,
+  totalPrimary: bigint,
+  totalSecondary: bigint,
+  totalLiquidity: bigint,
+): bigint {
+  if (totalPrimary + totalSecondary === 0n) {
+    return isqrt(addedPrimary * addedSecondary);
+  }
+
+  const ltA = (addedPrimary * totalLiquidity) / totalPrimary;
+  const ltB = (addedSecondary * totalLiquidity) / totalSecondary;
+  return ltA > ltB ? ltB : ltA;
```

Comment:
in smart contract there is assert that lt minted cannot be lesser or equal 0. Do we want to add such check here?